SIPS/sip-29.md

```diff
@@ -117,8 +117,10 @@ type Conversion = {
 
 type OnAssetsConversionArguments = {
   conversions: Conversion[];
+  includeMarketData?: boolean;
 };
 ```
+- `includeMarketData` - A boolean that indicates whether the Snap should include market data in the response. If `true`, the Snap SHOULD include market data in the response if available.
```

Comment:
What happens if it's `true`, but the Snap didn't include market data?

Reply:
Nothing special, it's okay if the Snap is not able to provide data if it's not able to. We will just not display any data in the asset details screen